modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java

```diff
@@ -1531,38 +1553,29 @@ private void _validateDeletionType(
 	}
 
 	private void _validateEdge(
-			boolean edge, ObjectRelationship objectRelationship)
+			boolean edge, ObjectDefinition objectDefinition1,
+			ObjectDefinition objectDefinition2, long objectRelationshipId,
+			String type)
 		throws PortalException {
 
 		if (!edge || !FeatureFlagManagerUtil.isEnabled("LPS-187142")) {
 			return;
 		}
 
-		if (!Objects.equals(
-				objectRelationship.getType(),
-				ObjectRelationshipConstants.TYPE_ONE_TO_MANY)) {
-
+		if (!type.equals(ObjectRelationshipConstants.TYPE_ONE_TO_MANY)) {
```

Comment:
Are you sure that `type` will not be Null at this point? Can this be null because the user did not send this information in the request?
Why did you change from` Objects.equals()`?

Reply:
I think it can be null, I switched it back to using `Objects.equals()`. Thank you